task3/maps/maps.go

```diff
@@ -0,0 +1,18 @@
+package maps
+
+import (
+	"sort"
+)
+
+func PrintSorted(mp map[int]string) []string {
+	sortedValues := make([]string, len(mp))
+	keys := make([]int, 0, len(mp))
+	for k := range mp {
+		keys = append(keys, k)
```

Comment:
Usually it used like 
```go
b :=make([]int, len(a), len(a))
for i, v:= range a {
   b[i] = someMagic(v)
}
```
This dummy example is invalid in terms of func, but shows the way to avoid unnecessarily usage of `append`, which is usually a costly overkill
upd: NWM, you have used this approach in `Reverse` func, so it's OK to keep this one as is for now,

Reply:
I find that "map entries have no index in Go; there's no way you can get an index out of any element. Also, each time you iterate on a map with range, you get a different order - another hint that there's no index concept in maps". We can get indexes like this; but is it right way?
```
	idx := 0
	for k, _ := range mp {
		fmt.Println(k, v)
		idx += 1
	}
```

Reply:
Exactly! Introducing a new int variable instead of a couple(~log2(len(mp)) of arrays created by append.
And yes, it's a nature of map - to be unordered data structure.
It's not a blocker anyway.